mu/logic.py

```diff
@@ -566,6 +566,15 @@ def load(self):
         """
         path = self._view.get_load_path(get_workspace_dir())
         logger.info('Loading script from: {}'.format(path))
+        # see if file is open first
+        for widget in self._view.widgets:
+            if widget.path is None:  # this widget is an unsaved buffer
+                continue
+            if path in widget.path:
+                self._view.show_message('Requested file is already open.')
```

Comment:
I changed the wording to "That file is already open"
The other suggestion-I was not sure how to express the filename to the user while being simple.